mod-notify/mod-notify.postman_collection.json

```diff
@@ -4807,29 +4651,29 @@
 							"listen": "test",
 							"script": {
 								"id": "926890c6-7d6e-4060-b6bf-c52ff73b5974",
-								"type": "text/javascript",
 								"exec": [
 									"pm.test(\"Status code is 404\", function () {",
 									"    pm.response.to.have.status(404);",
 									"});   ",
 									"    ",
 									"pm.test(\"Response contains correct data\", function () { ",
-									" pm.expect(pm.response.text()).to.equal(\"No suitable module found for path /notify/_username\");",
+									" pm.expect(pm.response.text()).to.equal(\"No suitable module found for path /notify/_username for tenant fs00000000\");",
```

Comment:
shouldn't we try to get the tenant value from the environment variable? Because API tests can be run against any tenant for a fact.